### PR TITLE
Run a force-bump with dev-preview2

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

**Comment by stuggi** [comment]
`Error: Project github.com/openstack-k8s-operators/designate-operator does not have the default branch master`
wondering why this is happening. designate is not yet integrated

**Comment by stuggi** [comment]
recheck

**Comment by rebtoor** [comment]
recheck

**Comment by rebtoor** [comment]
recheck

File: apis/go.mod [hunk 1]
```diff
@@ -4,24 +4,24 @@ go 1.19
...
+	github.com/openstack-k8s-operators/ironic-operator/api v0.2.0
+	github.com/openstack-k8s-operators/keystone-operator/api v0.2.0
+	github.com/openstack-k8s-operators/lib-common/modules/common v0.2.0
+	github.com/openstack-k8s-operators/manila-operator/api v0.2.0
+	github.com/openstack-k8s-operators/mariadb-operator/api v0.2.0
+	github.com/openstack-k8s-operators/neutron-operator/api v0.2.0
+	github.com/openstack-k8s-operators/nova-operator/api v0.3.0
+	github.com/openstack-k8s-operators/octavia-operator/api v0.2.0
+	github.com/openstack-k8s-operators/ovn-operator/api v0.2.0
+	github.com/openstack-k8s-operators/placement-operator/api v0.3.0
```
Comment (rebtoor) [inline apis/go.mod:22]: same as above

File: apis/go.mod [hunk 2]
```diff
@@ -4,24 +4,24 @@ go 1.19
...
+	github.com/openstack-k8s-operators/swift-operator/api v0.3.0
```
Comment (rebtoor) [inline apis/go.mod:23]: same as above

File: apis/go.mod [hunk 3]
```diff
@@ -65,8 +65,8 @@ require (
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
 	github.com/openshift/api v3.9.0+incompatible // indirect
-	github.com/openstack-k8s-operators/lib-common/modules/openstack v0.1.1-0.20230927082538-4f614f333d17 //indirect
-	github.com/openstack-k8s-operators/lib-common/modules/storage v0.1.1-0.20230927082538-4f614f333d17
+	github.com/openstack-k8s-operators/lib-common/modules/openstack v0.1.1-0.20231001084618-12369665b166 //indirect
```
Comment (rebtoor) [inline apis/go.mod:68]: `v0.2.0` ?

File: apis/go.mod [hunk 4]
```diff
@@ -4,24 +4,24 @@ go 1.19
...
-	github.com/openstack-k8s-operators/mariadb-operator/api v0.1.1-0.20230928152002-65395552e015
-	github.com/openstack-k8s-operators/neutron-operator/api v0.1.1-0.20230927135156-a3209100b1f3
-	github.com/openstack-k8s-operators/nova-operator/api v0.1.2-0.20230928160649-87342575139b
-	github.com/openstack-k8s-operators/octavia-operator/api v0.0.0-20230928114327-f032acaab883
-	github.com/openstack-k8s-operators/ovn-operator/api v0.1.1-0.20230927181029-2b25ed78973d
-	github.com/openstack-k8s-operators/placement-operator/api v0.1.1-0.20230927115929-d73f6b7ba241
-	github.com/openstack-k8s-operators/swift-operator/api v0.1.1-0.20230928061421-52a9feee12c9
-	github.com/openstack-k8s-operators/telemetry-operator/api v0.1.1-0.20230927140521-cf7134b9d873
+	github.com/onsi/gomega v1.28.0
+	github.com/openstack-k8s-operators/cinder-operator/api v0.3.0
```
Comment (rebtoor) [inline apis/go.mod:8]: Should we call `v0.2.0` here ?

File: apis/go.mod [hunk 5]
```diff
@@ -4,24 +4,24 @@ go 1.19
...
+	github.com/openstack-k8s-operators/heat-operator/api v0.2.0
+	github.com/openstack-k8s-operators/horizon-operator/api v0.2.0
+	github.com/openstack-k8s-operators/infra-operator/apis v0.2.0
+	github.com/openstack-k8s-operators/ironic-operator/api v0.2.0
+	github.com/openstack-k8s-operators/keystone-operator/api v0.2.0
+	github.com/openstack-k8s-operators/lib-common/modules/common v0.2.0
+	github.com/openstack-k8s-operators/manila-operator/api v0.2.0
+	github.com/openstack-k8s-operators/mariadb-operator/api v0.2.0
+	github.com/openstack-k8s-operators/neutron-operator/api v0.2.0
+	github.com/openstack-k8s-operators/nova-operator/api v0.3.0
```
Comment (rebtoor) [inline apis/go.mod:19]: same as above

File: go.mod [hunk 6]
```diff
@@ -7,28 +7,28 @@ require (
...
-	github.com/openstack-k8s-operators/lib-common/modules/common v0.1.1-0.20230927082538-4f614f333d17
-	github.com/openstack-k8s-operators/manila-operator/api v0.1.1-0.20230928195132-c19d2f673d7d
-	github.com/openstack-k8s-operators/mariadb-operator/api v0.1.1-0.20230928152002-65395552e015
-	github.com/openstack-k8s-operators/neutron-operator/api v0.1.1-0.20230927135156-a3209100b1f3
-	github.com/openstack-k8s-operators/nova-operator/api v0.1.2-0.20230928160649-87342575139b
-	github.com/openstack-k8s-operators/octavia-operator/api v0.0.0-20230928114327-f032acaab883
-	github.com/openstack-k8s-operators/openstack-ansibleee-operator/api v0.1.1-0.20230928203449-96a16f009b6d
-	github.com/openstack-k8s-operators/openstack-baremetal-operator/api v0.1.1-0.20230927091601-9064ac2029e5
+	github.com/onsi/gomega v1.28.0
+	github.com/openstack-k8s-operators/cinder-operator/api v0.3.0
```
Comment (rebtoor) [inline go.mod:11]: `v0.2.0` ?

File: go.mod [hunk 7]
```diff
@@ -7,28 +7,28 @@ require (
...
+	github.com/openstack-k8s-operators/heat-operator/api v0.2.0
+	github.com/openstack-k8s-operators/horizon-operator/api v0.2.0
+	github.com/openstack-k8s-operators/infra-operator/apis v0.2.0
+	github.com/openstack-k8s-operators/ironic-operator/api v0.2.0
+	github.com/openstack-k8s-operators/keystone-operator/api v0.2.0
+	github.com/openstack-k8s-operators/lib-common/modules/common v0.2.0
+	github.com/openstack-k8s-operators/manila-operator/api v0.2.0
+	github.com/openstack-k8s-operators/mariadb-operator/api v0.2.0
+	github.com/openstack-k8s-operators/neutron-operator/api v0.2.0
+	github.com/openstack-k8s-operators/nova-operator/api v0.3.0
```
Comment (rebtoor) [inline go.mod:23]: `v0.2.0` ?

File: go.mod [hunk 8]
```diff
@@ -7,28 +7,28 @@ require (
...
+	github.com/openstack-k8s-operators/octavia-operator/api v0.2.0
+	github.com/openstack-k8s-operators/openstack-ansibleee-operator/api v0.2.0
+	github.com/openstack-k8s-operators/openstack-baremetal-operator/api v0.3.1-0.20231005142619-75b1fbd8777c
```
Comment (rebtoor) [inline go.mod:26]: `v0.2.0` ?

File: go.mod [hunk 9]
```diff
@@ -7,28 +7,28 @@ require (
...
 	github.com/openstack-k8s-operators/openstack-operator/apis v0.0.0-20230725141229-4ce90d0120fd
-	github.com/openstack-k8s-operators/ovn-operator/api v0.1.1-0.20230927181029-2b25ed78973d
-	github.com/openstack-k8s-operators/placement-operator/api v0.1.1-0.20230927115929-d73f6b7ba241
-	github.com/openstack-k8s-operators/swift-operator/api v0.1.1-0.20230928061421-52a9feee12c9
-	github.com/openstack-k8s-operators/telemetry-operator/api v0.1.1-0.20230927140521-cf7134b9d873
+	github.com/openstack-k8s-operators/ovn-operator/api v0.2.0
+	github.com/openstack-k8s-operators/placement-operator/api v0.3.0
```
Comment (rebtoor) [inline go.mod:29]: `v0.2.0`?

File: go.mod [hunk 10]
```diff
@@ -7,28 +7,28 @@ require (
...
+	github.com/openstack-k8s-operators/swift-operator/api v0.3.0
```
Comment (rebtoor) [inline go.mod:30]: `v0.2.0`?

File: go.mod [hunk 11]
```diff
@@ -75,8 +75,8 @@ require (
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
 	github.com/openshift/api v3.9.0+incompatible
-	github.com/openstack-k8s-operators/lib-common/modules/openstack v0.1.1-0.20230927082538-4f614f333d17 //indirect
-	github.com/openstack-k8s-operators/lib-common/modules/storage v0.1.1-0.20230927082538-4f614f333d17 //indirect
+	github.com/openstack-k8s-operators/lib-common/modules/openstack v0.1.1-0.20231001084618-12369665b166 //indirect
```
Comment (rebtoor) [inline go.mod:78]: `v0.2.0`?

**Review by rebtoor — CHANGES_REQUESTED** [review]

**Pull request merged**